rbac/management/utils.py

```diff
@@ -122,9 +122,9 @@ def get_principal(
 
 def verify_principal_with_proxy(username, request, verify_principal=True):
     """Verify username through the BOP."""
-    org_id = request.user.org_id
-    proxy = PrincipalProxy()
     if verify_principal:
+        org_id = request.user.org_id
```

Comment:
bumping the check up just because the way I am using this method does not have `request.user.org_id` set

I am using `verify_principal=False` so this check doesn't run anyways